wordpress-mobile/WordPress-Android
# Update screenshots to new colour scheme

**To Test**

0. Check out this branch. Ensure that all the tests pass (this will tell us if the changes here are compatible with the existing UI tests).
1. Ensure you've installed the complete Noto Sans font family on your local machine: https://fonts.google.com/specimen/Noto+Sans.
2. Generate a set of Android screenshots using `bundle exec fastlane android screenshots`. This will take about an hour and a half. The content should all load instantly, using the mocked responses.
3. Ensure that the `fastlane/metadata` directory contains the new screenshots.
4. Run `bundle exec fastlane android create_promo_screenshots`. It'll generate localized screenshots into the fastlane/promo_screenshots directory.
5. The screenshots should match those currently used as experiments in the Google Play Store.
